scripts/generate_export.py

```diff
@@ -1,12 +1,16 @@
 import os
 import sys
 import time
+from indico.client.client import IndicoClient
```

Comment:
the indico_wrapper object handles this sort of stuff now. Will comment on the rest of these with more details.